src/pymovements/dataset/dataset.py

```diff
@@ -40,6 +41,10 @@
 from pymovements.reading_measures import ReadingMeasures
 
 
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
After some reconsideration I moved the disclaimer to `dataset.py`, similar to your original proposal, but returning a string instead.

The main reason was, that `logger.info` has `dataset/dataset_download.py` as a source. I plan to rename this module to `dataset/_download.py` and then it would just be strange to present an info source from there. `dataset/dataset.py` as a info source would be much more comprehensible by a user.